core/dyn-configs/src/lib.rs

```diff
@@ -46,7 +46,6 @@ impl UpdateableConfigLoader {
         updateable_configs: UpdateableConfigs,
         tx: Sender<Result<UpdateableConfigs, Arc<UpdateableConfigLoaderError>>>,
     ) -> Self {
-        near_o11y::reload_log_config(updateable_configs.log_config.as_ref());
```

Comment:
Unrelated to this PR, but this duplication is annoying albeit harmless. The reload is done at line 51.